jetbrains-core/tst/software/aws/toolkits/jetbrains/services/cloudformation/yaml/YamlCloudFormationTemplateTest.kt

```diff
@@ -124,10 +122,10 @@ Parameters:
         runInEdtAndWait {
             assertThat(template.parameters().toList()).hasSize(2)
             val tableTag = template.parameters().firstOrNull { it.logicalName == "TableTag" }
-            assertNotNull(tableTag)
-            assertNull(tableTag.defaultValue())
-            assertNotNull(tableTag.description())
-            assertNull(tableTag.constraintDescription())
+            assertThat(tableTag).isNotNull
+            assertThat(tableTag!!.defaultValue()).isNull()
```

Comment:
yeah this is the only problem with using AssertJ is you don't get the smart-typing. I'd prefer these were re-written using `software.aws.toolkits.core.utils.test.notNull`

```
assertThat(tableTag).notNull.satisfies { tag ->
  assertThat(tag.defaultValue()).isNull()
  assertThat(tag.description()).isNotNull()
  assertThat(tag.constraintDescription()).isNull()
}
```

Reply:
Sometimes this doesn't work (mostly due to tests being too complicated), so unbanned the kotlin test assertNotNull as it handles nullability properly